CollectionData/2020_weeks.csv

```diff
@@ -0,0 +1,52 @@
+2020,	52
```

Comment:
do we really need to store this data this way? i feel we could generate this programmatically on the fly

Reply:
This is just a template for one year (52 weeks) that we can swap with a text editor to go back till 1976..
Note: The average grant file size is .5 GB and runs for about 5 mins while the application files are larger and takes more time. The Idea us not to run all requests at the same time to avoid extra charges.